feature/experimental/bgp/ate_tests/base_bgp_session_parameters/base_bgp_session_parameters_test.go

```diff
@@ -359,7 +426,7 @@ func TestParameters(t *testing.T) {
 			tc.ateConf.Push(t)
 			tc.ateConf.StartProtocols(t)
 			t.Log("Verify BGP session state : ESTABLISHED")
-			gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*100, oc.Bgp_Neighbor_SessionState_ESTABLISHED)
+			gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*30, oc.Bgp_Neighbor_SessionState_ESTABLISHED)
```

Comment:
Revert the await timeout reduction here also?

Reply:
updated, thank you